src/pages/Home/StoreLocator.js

```diff
@@ -23,83 +34,69 @@ const StoreList = styled.div`
   flex-wrap: wrap;
   align-items: center;
   justify-content: space-evenly;
-  width: 90%;
+  width: 100%;
   max-width: 450px;
-  margin: 0 auto;
+  margin: auto;
+
+  @media ${device.mobile} {
+    text-align: center;
+    align-content: center;
+    margin: auto;
+  }
   @media ${device.tablet} {
     flex-direction: column;
-    max-width: '';
-    margin: 0 auto 0 20px;
-    width: 60%;
+    text-align: center;
   }
   @media ${device.laptop} {
-    margin: 0 auto 0 40px;
+    text-align: left;
+    padding-left: 1rem;
   }
 `;
 
 export const stores = [
```

Comment:
You had mentioned wanting to put this into a separate file. I'm in support of that if you want to, but no pressure!

Reply:
@dbridenbeck that will take longer than I have time for today so I will push the other changes and come back to this one

Reply:
sounds great!